diff --git a/codecs_v2/audio/gsm_amr/amr_wb/dec/src/homing_amr_wb_dec.cpp b/codecs_v2/audio/gsm_amr/amr_wb/dec/src/homing_amr_wb_dec.cpp
index 59c6c0a..9b4a870 100644
--- a/codecs_v2/audio/gsm_amr/amr_wb/dec/src/homing_amr_wb_dec.cpp
+++ b/codecs_v2/audio/gsm_amr/amr_wb/dec/src/homing_amr_wb_dec.cpp
@@ -134,7 +134,7 @@ extern "C"
 ; LOCAL STORE/BUFFER/POINTER DEFINITIONS
 ; Variable declaration - defined here and used outside this module
 ----------------------------------------------------------------------------*/
-const int16 prmnofsf[NUM_OF_SPMODES] =
+static const int16 prmnofsf[NUM_OF_SPMODES] =
 {
     63,  81, 100,
     108, 116, 128,
diff --git a/codecs_v2/audio/gsm_amr/amr_wb/dec/src/isp_isf.cpp b/codecs_v2/audio/gsm_amr/amr_wb/dec/src/isp_isf.cpp
index 41db7e3..0552733 100644
--- a/codecs_v2/audio/gsm_amr/amr_wb/dec/src/isp_isf.cpp
+++ b/codecs_v2/audio/gsm_amr/amr_wb/dec/src/isp_isf.cpp
@@ -108,7 +108,7 @@ terms listed above has been obtained from the copyright holder.
 
 /* table of cos(x) in Q15 */
 
-const int16 table[129] =
+static const int16 table[129] =
 {
     32767,
     32758,  32729,  32679,  32610,  32522,  32413,  32286,  32138,
diff --git a/codecs_v2/audio/gsm_amr/amr_wb/dec/src/phase_dispersion.cpp b/codecs_v2/audio/gsm_amr/amr_wb/dec/src/phase_dispersion.cpp
index f90a534..7b08a40 100644
--- a/codecs_v2/audio/gsm_amr/amr_wb/dec/src/phase_dispersion.cpp
+++ b/codecs_v2/audio/gsm_amr/amr_wb/dec/src/phase_dispersion.cpp
@@ -109,7 +109,7 @@ terms listed above has been obtained from the copyright holder.
 /* impulse response with phase dispersion */
 
 /* 2.0 - 6.4 kHz phase dispersion */
-const int16 ph_imp_low[L_SUBFR] =
+static const int16 ph_imp_low[L_SUBFR] =
 {
     20182,  9693,  3270, -3437, 2864, -5240,  1589, -1357,
     600,  3893, -1497,  -698, 1203, -5249,  1199,  5371,
@@ -122,7 +122,7 @@ const int16 ph_imp_low[L_SUBFR] =
 };
 
 /* 3.2 - 6.4 kHz phase dispersion */
-const int16 ph_imp_mid[L_SUBFR] =
+static const int16 ph_imp_mid[L_SUBFR] =
 {
     24098, 10460, -5263,  -763,  2048,  -927,  1753, -3323,
     2212,   652, -2146,  2487, -3539,  4109, -2107,  -374,
